libgraph/src/PropertyGraph.cpp

```diff
@@ -113,15 +113,15 @@ WriteEntityTypeIDsArray(
       ff->Write(&data, sizeof(katana::EntityTypeIDArrayHeader));
 
   if (!aro_sts.ok()) {
-    return katana::ArrowToTsuba(aro_sts.code());
+    return katana::ArrowToKatana(aro_sts.code());
```

Comment:
This might not be useful anymore since `KATANA_CHECKED` can just handle Arrow status objects. We can leave that bit as future work though I totally get wanting this PR to be pure refactor and not change error messages or anything like that.

Reply:
Thank you and I agree. I don't want to make any "real changes" in this PR given that the changeset is so big. I will make note of the points you raised and address them in the near future. I'm planning a subsequent PR that moves `EntityManager` and parallelizes it.